### DISCUSSION TITLE
The codespace is not available.

### DISCUSSION THREAD

**Discussion by jeremysf** [community/community#5077] [Codespaces] [answered]
My codespace will work for a bit then eventually display the error "The codespace is not available." and no amount of refreshing beyond that will help.

I see this used to be a common issue, what are the right steps to figure out how to fix? I can't seem to see a way to get any logs, etc.

Oh, and now on refreshing (maybe an hour after the problem started), I just get "Starting..." indefinitely.

**Comment by tanmayeekamath** [comment, marked as answer]
Hey @jeremysf, thanks for reaching out. Can you please file a support ticket by going to: https://support.github.com/

Since we don't have any known issues, this will help our engineering team get more details on your instance and help investigate the issue.

**Reply by TechKemon** [reply]
same issue here. my github id is TechKemon
